packages/transformers/react-refresh-babel/src/ReactRefreshBabelTransformer.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import semver from 'semver';
+import invariant from 'assert';
+import {Transformer} from '@parcel/plugin';
+import {relativeUrl} from '@parcel/utils';
+import SourceMap from '@parcel/source-map';
+import {transformFromAst} from '@babel/core';
+import generate from '@babel/generator';
+import {parse} from '@babel/parser';
+
+function shouldExclude(asset, options) {
+  return (
+    !asset.env.isBrowser() ||
+    !options.hot ||
+    !asset.isSource ||
+    asset.filePath.includes('packages/runtimes')
+  );
+}
+
+export default new Transformer({
+  async transform({asset, options}) {
+    // TODO tests break with this?
+    //    asset.type = 'js';
```

Comment:
🤷‍♂

Reply:
makes sense. the JS transformer should handle this